Tasks/VsTest/task.json

```diff
@@ -46,12 +47,25 @@
     ],
     "inputs": [
         {
-            "name": "testAssembly",
-            "type": "string",
+            "name": "testAssemblyVer2",
+            "type": "multiLine",
             "label": "Test Assembly",
-            "defaultValue": "**\\*test*.dll;-:**\\obj\\**",
+            "defaultValue": "**\\*test*.dll\n!**\\obj\\**",
             "required": true,
-            "helpMarkDown": "Test binaries to run tests on.  Wildcards can be used.  For example, `**\\*test*.dll;-:**\\obj\\**` for all dlls with test in name while excluding files in any sub-directory named obj.",
+            "helpMarkDown": "Run tests from the specified files. Supports multiple lines of minimatch patterns. [More Information](https://go.microsoft.com/fwlink/?LinkId=835764)",
+            "groupName": "executionOptions",
+            "properties": {
+                "rows": "3",
+                "resizable": "true"
+            }
+        },
+        {
+            "name": "searchFolder",
+            "type": "string",
+            "label": "Search Folder",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Folder to search for the test assemblies. Defaults to $(System.DefaultWorkingDirectory) for build and $(System.ArtifactsDirectory) for release.",
```

Comment:
i think system.defaultworkingdirectory is the same as system.artifactsdirectory for a release